srcs/init/variable.c

```diff
@@ -0,0 +1,71 @@
+#include "shell_env.h"
+#include "utils.h"
+
+t_variable *create_variable(char *name, char *value, bool exported)
+{
+	t_variable *v;
+
+	v = (t_variable *)malloc_or_die(sizeof(t_variable));
```

Comment:
The cast allows for pre-1989 versions of malloc that originally returned a char *.